shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -81,6 +84,9 @@
     private final BroadcastReceiver mDiscoveryReceiver;
     private final List<ActivityLifecycleListener> mActivityLifecycleListeners;
     private long mNativePlatformView;
+    private boolean isSoftware = false; //using the software renderer or not
+
+    private Bitmap bmap;
```

Comment:
Declare "bmap" as volatile, and do not assign it until after the new bitmap's copyPixelsFromBuffer completes.  This will ensure that all writes performed to the bitmap on the GPU thread are subsequently seen when the platform thread reads "bmap";